### PR TITLE
add assignment-4

### PR DESCRIPTION
**Pull request by mharshita**
Add Assignment 4

### CODE REVIEW

**Comment by vickydarlinn** [comment]
nice work!!!

**Comment by sumathi3399** [comment]
Looks Good 👍

**Review by sumathi3399 — APPROVED** [review]

File: src/Card.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+import { FaUser } from "react-icons/fa";
+
+const Card = (props) => {
+  return (
+    <div className="card">
+      <div className="card-body">
+        <div className="img">
+          <FaUser size="5em" className="user-icon-card" />
+        </div>
+        <div className="card-info">
```
Comment (ap221882) [inline src/Card.js:11]: You can destructure props right away in the argument of the Card component.

**Review by ap221882 — COMMENTED** [review]
Nicely written. Try component-level styling, it makes CSS/styles more readable.

File: src/Card.js (hunk 1 above)
Comment (mharshita) [reply src/Card.js:11]: Sure! Will make changes in the next PR. Thanks for the review!

**Pull request merged**